src/generator/index.tsx

```diff
@@ -95,21 +88,19 @@ export default async function render({ messages, channel, callbacks, ...options
           minHeight: '100vh',
         }}
       >
-        <DiscordMessages messages={messages} channel={channel} callbacks={callbacks} {...options} />
+        {transcriptContent}
```

Comment:
Is there a reason you've moved the DiscordMessages outside of this JSX block into it's own await statement? In my opinion, this is less readable.